ncj/maya/render-default-linux/job.template.json

```diff
@@ -17,6 +17,17 @@
                 "description": "The unique name of the job."
             }
         },
+        "mayaRenderer": {
```

Comment:
Version

Reply:
yep


---
In reply to: [191116090](https://github.com/Azure/BatchLabs-data/pull/16#discussion_r191116090) [](ancestors = 191116090)